app/main.py

```diff
@@ -1,2 +1,57 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.items = []
+        self.capacity = 8
+        self.hash_table = [[] for _ in range(self.capacity)]
+
+    def __setitem__(self, key, value):
+        self.items.append([key, value])
+        threshold = int((self.capacity * 2 / 3))
+        size = len(self.items)
+        if size == threshold:
+            self.capacity *= 2
+            self.hash_table = [[] for _ in range(self.capacity)]
+            for element in self.items:
+                self.hashing(hash(element[0]) % self.capacity,
+                             element[1],
+                             element[0])
+            return
+
+        hash_ = hash(key) % self.capacity
+        self.hashing(hash_, value, key)
+
+    def __getitem__(self, item):
+        n = 0
+        while True:
+            if len(self.hash_table[n]) > 0:
+                if self.hash_table[n][0] == item:
+                    return self.hash_table[n][1]
+            n += 1
```

Comment:
It can be much more concise. It is relevant to use such incrementation: `index = (index + 1) % self.capacity`.
It is possible to write this block within one while loop in 5 lines.